lib/logitech_receiver/base_usb.py

```diff
@@ -26,11 +26,29 @@
 _DRIVER = ('hid-generic', 'generic-usb', 'logitech-djreceiver')
 
 
-# each tuple contains (vendor_id, product_id, usb interface number, hid driver)
-_unifying_receiver = lambda product_id: (0x046d, product_id, 2, _DRIVER)
-_nano_receiver = lambda product_id: (0x046d, product_id, 1, _DRIVER)
-_lenovo_receiver = lambda product_id: (0x17ef, product_id, 1, _DRIVER)
-_lightspeed_receiver = lambda product_id: (0x046d, product_id, 2, _DRIVER)
+_unifying_receiver = lambda product_id: {'vendor_id':0x046d,
```

Comment:
nitpick: can we indent it like this?
```
_unifying_receiver = lambda product_id: {
	'vendor_id':0x046d,
	'product_id':product_id, 
	'usb_interface':2,
	'hid_driver':_DRIVER,
	'name':'Unifying Receiver'
}
```

Reply:
Done.  I did modified the commit and force-pushed the result.